src/test/anovos/data_transformer/test_transformers.py

```diff
@@ -0,0 +1,184 @@
+import os
+import pytest
+from pyspark.sql import functions as F
+from anovos.data_ingest.data_ingest import *
+from anovos.data_transformer.transformers import *
+
+sample_parquet = "./data/test_dataset/part-00001-3eb0f7bb-05c2-46ec-8913-23ba231d2734-c000.snappy.parquet"
+
+@pytest.mark.usefixtures("spark_session")
```

Comment:
This line is not necessary, you're not using it below either.

Reply:
Yes, I have removed it.